elements/lisk-validator/src/keywords/field_number.ts

```diff
@@ -48,26 +42,22 @@ const deepValue = (obj: object, path: string): any => {
 	return result;
 };
 
-interface AjvContext {
-	root: {
-		schema: object;
-	};
-	schemaPath: string;
-}
-
 const compile = (
-	value: number,
-	parentSchema: object,
-	it: Partial<AjvContext>,
-): ValidateFunction => {
+	value: JSONSchemaType<number>,
+	parentSchema: AnySchemaObject,
+	it: SchemaCxt,
+): DataValidateFunction => {
```

Comment:
I think no need to define the return type if the type is not exposed

Reply:
Tried that it was complaining about not matching type with compile function.

Reply:
if above is type def can be used, this isn't a problem